cttportfolio.css

```
@@ -3,168 +3,481 @@
-}
-.mainprojectlink{
+	height:50%;
 	display:flex;
+	justify-content:space-between;
+}
+.project2 {
+	align-items:flex-end;
+	margin-top:100px;
 }
-.mainprojectlink > *{
+.project_container > * {
+	height:100%;
+}
+.project_img{
 	display:block;
-	width:50%;
-	height:50%;
-
+	height:100%;
+	max-width:100%;
+	object-fit:cover;
+	
 }
-.subportfolio{
-	background-color:#00bfff;
+.pf_link{
+	width:30%;
+	object-fit:cover;
+}
+.project_text{
+	text-decoration:none;
+	color:rgb(2,7,9);
+	font-size:2rem;
+	width:70%;
 	display:flex;
+	flex-direction:column;
 	justify-content:space-between;
+	border-radius:10px;
+}
+.project_text > *{
+	padding:20px;
+}
+.project_text h5{
+	color:rgb(248,248,255);
+	font-size:4rem;
+	font-weight:600;
+	text-transform:uppercase;
+    -webkit-text-stroke-width: 0.3px;
+    -webkit-text-stroke-color: black;
+	margin:0;
+	align-self:flex-start;
 }
-.subportfolio > a{
-	width:33%;
-	height:33%;
+.project1 .project_text{
+	margin-left:50px;
 }
-.subprojectlink > *{
-	width:100%;
+.project2 .project_text{
+	margin-right:50px;
+}
+.project_main_links{
+	width:70%;
+	align-self:flex-end;
+	display:flex;
+	justify-content:space-between;
+	padding-bottom:2%;
+}
+.project_main_links .project_icon_size{
+	width:8%;
+}
+.pf_top button{
+	margin-top:80px;
+	margin-bottom:80px;
+}
+/*======================================== SUB PORTFOLIO SECTION ========================================*/
+.pf_sub{
+	display:flex;
+	justify-content:space-between;
+	height:500px;
+}
+.pf_sub > a {
+	width:30%;
+	height:100%;
+	text-decoration:none;	
+}
+.pf_sub_link_div{
 	height:100%;
 	display:flex;
 	flex-direction:column;
 	align-items:center;
 }
-.mainprojectlink > img{
+.pf_sub_link_div > img{
 	object-fit:cover;
+	height:100%;
+	max-width:100%;
 }
-a.portfoliolinks:hover{
-	border:1px solid black;	
+.pf_sub h6{
+	text-decoration:none;
+	font-weight:600;
+	font-size:2.0rem;
+	padding:5%;
 }
-
 /*======================================== CONTACT SECTION ========================================*/
 .contactme{
-	background-color:red;
+	display:flex;
+	align-items:center;
+	flex-direction:column;
+	margin-top:100px;
 }
 .contactmetext{
 	display:flex;
+	flex-direction:column;
 	justify-content:space-between;
-	padding:0 10%;
+	align-items:center;
+	width:100%;
+}
+.contactmetext > div{
+	margin:50px 0px;
+	padding:50px 50px;
+}
+.contact_div{
+	display:flex;
+	justify-content:space-between;
+	align-items:center;
+	width:80%;
+}
+.contact_div > *{
+	margin:0;
+}
+.contact_div h6{
+	font-size:2rem;
+}
+.contact_input{
+	width:50%;
+}
+.contact_input {
+	margin-top:50px;
+	display:flex;
+}
+.contact_input input{
+	margin-right:50px;
+	width:100%;
+	font-size:2rem;
+	padding-left:10px;
+	border-radius:10px;
+}
+.contact_input button{
+	box-shadow: 0px -2px 5px rgba(248,248,255, 0.0)
+}
+.contact_input button:hover{
+	border:1px solid rgba(248,248,255,0.6);
+	box-shadow: 0px -2px 5px rgba(248,248,255, 0.6)
+}
+.contact_div > h6{
+	font-size:2.0rem;
```

You don't need `.0` here.
```suggestion
	font-size:2rem;
```